lib/govuk_publishing_components/config.rb

```diff
@@ -28,5 +28,87 @@ def self.component_directory_name
     def self.gem_directory
       Gem::Specification.find_by_name("govuk_publishing_components").gem_dir
     end
+
+    ALL_ASSETS = {
```

Comment:
Thanks, good suggestion. I've changed `ALL_ASSETS ` to `ALL_STYLESHEETS` and `all_assets ` to `all_stylesheets` (since the method is used like so: `GovukPublishingComponents::Config.all_stylesheets`).

I've also changed `app_stylesheets` to `APP_STYLESHEETS` since I think it aligns better with the convention of using uppercase for constants.

I've also updated https://github.com/alphagov/email-alert-frontend/pull/1655/commits/db259d5eb2bbcd9ed8f8486fecaacb1a0684db38 and https://github.com/alphagov/govuk-developer-docs/pull/4216/commits/f574145a097abaed639fbdde5d29b70e0474e20d

Reply:
Nice, thanks for updating, the name change makes sense in the context of where the method is called from 👍